<launch>
    <node pkg="human_aware_robot_navigation" type="main.py" name="main"/>
</launch>
